### PR TITLE
fix: updates base url of website

### PR DESCRIPTION
**Pull request by nilekhc**
Signed-off-by: Nilekh Chaudhari <1626598+nilekhc@users.noreply.github.com>

### CODE REVIEW

**Comment by nilekhc** [comment]
cc @maxsmythe @sozercan @ritazh

File: website/static/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (sozercan) [inline website/static/index.html:1]: unless this gets copied over to root of `gh-pages` branch automatically? it will need to be a separate pr to `gh-pages` branch

Comment (nilekhc) [reply website/static/index.html:1]: `yarn build` takes care of it.

**Review by sozercan — APPROVED** [review]
